src/adapter/src/error.rs

```diff
@@ -479,6 +487,12 @@ impl fmt::Display for AdapterError {
                 f,
                 "cluster replica '{cluster_name}.{replica_name}' does not exist"
             ),
+            AdapterError::UnknownCluster { cluster_name } => {
```

Comment:
Wait why are these new? These error paths already existed during sequence planning?

Reply:
The existing error paths present in the planner and the errors themselves come from the catalog: https://github.com/MaterializeInc/materialize/blob/00cb35aeb3c00f2844e09797db4a2bce33a648d3/src/sql/src/catalog.rs#L752-L757

I'm going to delete these now that we'll just re-run the whole command when the catalog changes.